sportsref/ncaaf/players.py

```diff
@@ -147,8 +152,17 @@ def rushing_and_receiving(self):
         :returns: Pandas DataFrame with stats.
         """
         doc = self.getDoc()
-        table = doc('#rushing')
+        table = doc('#all_rushing')
```

Comment:
The new utils.get_html function takes care of this (just strips literally all comments). I realize that stripping all comments could be an issue, but for now it works. I actually think I may rewrite the relevant parts of the library using lxml instead of pyquery, because I've seen some benchmarks that it's a bit faster and I've been meaning to learn XPath. Anyways, for now, the new `utils.get_html` handles this, so using `table#rushing` should be fine.